lib/device.js

```diff
@@ -1137,6 +1277,53 @@ const async = require('async'),
         }
     }
 
+    function makeCSVOutputPath(options, next) {
+        const csvFormat = "csv"; // csv is default format
+
+        if (options.outputPath === null) {
+            options.fileName = createDateFileName(null, csvFormat);
+            options.destinationPath = path.resolve('.');
+        } else {
+            // get directory path, file path, file Extenstion from given path
+            const parseDirPath = path.dirname(options.outputPath),
+                parseBase = path.parse(options.outputPath).base;
+
+            let parseExt = path.parse(options.outputPath).ext;
+            parseExt = parseExt.split('.').pop();
+
+            log.info("device#captureScreen()#_makeCaptureOption()" + "dir name:" + parseDirPath
+                + " ,file name:" + parseBase + " ,inputFormat:" + parseExt);
+
+            // if specific path is given
+            if (parseBase) {
+                // parseBase is [filename].[ext] format
+                if (parseExt) {
```

Comment:
I think it can be deleted... right?

Reply:
I deleted the logic. Please check the changed logic again.